build.sbt

```
@@ -33,6 +34,15 @@ lazy val `portable-scala-reflect` = crossProject(JSPlatform, JVMPlatform, Native
 
     mimaPreviousArtifacts +=
       organization.value %%% moduleName.value % previousVersion,
+    mimaBinaryIssueFilters ++= List(
+      ProblemFilters.exclude[DirectMissingMethodProblem]("org.portablescala.reflect.Reflect.lookupInstantiatableClass_impl"),
+      ProblemFilters.exclude[DirectMissingMethodProblem]("org.portablescala.reflect.Reflect.lookupLoadableModuleClass_impl"),
+      ProblemFilters.exclude[MissingClassProblem]("org.portablescala.reflect.Reflect$MacroCompat$"),
+      ProblemFilters.exclude[MissingClassProblem]("org.portablescala.reflect.Reflect$MacroCompat$Scope1$"),
+      ProblemFilters.exclude[MissingClassProblem]("org.portablescala.reflect.Reflect$MacroCompat$Scope1$blackbox$"),
+      ProblemFilters.exclude[MissingClassProblem]("org.portablescala.reflect.Reflect$MacroCompat$Scope2$"),
+      ProblemFilters.exclude[MissingClassProblem]("org.portablescala.reflect.Reflect$MacroCompat$Scope2$Inner$"),
+    ),
```

We should add comments to tell why those exclusions are OK. Also, we can leverage wildcards:
```suggestion
    mimaBinaryIssueFilters ++= List(
      // Macro implementations, not an issue
      ProblemFilters.exclude[DirectMissingMethodProblem]("org.portablescala.reflect.Reflect.lookupInstantiatableClass_impl"),
      ProblemFilters.exclude[DirectMissingMethodProblem]("org.portablescala.reflect.Reflect.lookupLoadableModuleClass_impl"),
      // Private, not an issue
      ProblemFilters.exclude[MissingClassProblem]("org.portablescala.reflect.Reflect$MacroCompat$*"),
    ),
```

---

done :heavy_check_mark: